ross/results.py

```diff
@@ -1458,38 +1459,26 @@ def plot_magnitude(
             The figure object with the plot.
         """
         frequency_range = Q_(self.speed_range, "rad/s").to(frequency_units).m
-        magnitude = self.magnitude
-        number_dof = self.rotor.number_dof
 
         if fig is None:
             fig = go.Figure()
 
         for i, p in enumerate(probe):
-            dofx = p[0] * number_dof
-            dofy = p[0] * number_dof + 1
             angle = Q_(p[1], probe_units).to("rad").m
-
-            # fmt: off
-            operator = np.array(
-                [[np.cos(angle), - np.sin(angle)],
-                 [np.cos(angle), + np.sin(angle)]]
-            )
-
-            probe_resp = operator @ np.vstack((magnitude[dofx], magnitude[dofy]))
-            z = np.sqrt((probe_resp[0] * np.cos(angle)) ** 2 +
-                        (probe_resp[1] * np.sin(angle)) ** 2)
-            # fmt: on
-
-            z = Q_(z, "m").to(amplitude_units).m
+            vector = self._calculate_major_axis_per_node(node=p[0], angle=angle)
+            try:
+                probe_tag = p[2] + f" - Node {p[0]}"
```

Comment:
If the user wants to give a specific tag to the probe I think it is not a good idea to modify it, so I would remove the `f" - Node {p[0]}"` part, since this could be added by the user.

Reply:
Done in commit [4b3586b](https://github.com/ross-rotordynamics/ross/pull/716/commits/4b3586b3fcd42c2d2732fb1d9d0f46e035a29f8c).